lib/helper/Puppeteer.js

```diff
@@ -1090,7 +1093,8 @@ class Puppeteer extends Helper {
     for (const modifier of modifiers) {
       await this.page.keyboard.down(modifier);
     }
-    await this.page.keyboard.press(key);
+    if (keyDefinitions[key]) await this.page.keyboard.press(key);
```

Comment:
This will be most likely always `true` as `getNormalizedKey` returns key definitions.